ios/bottom-tabs/RNSBottomTabsHostComponentView.mm

```diff
@@ -194,6 +194,25 @@ - (void)invalidate
 
 #endif
 
+#pragma mark - RNSContentScrollViewProviding
+
+- (UIScrollView *)findContentScrollView
+{
+#if !TARGET_OS_TV && !TARGET_OS_VISION
+  if (_controller.selectedViewController == _controller.moreNavigationController) {
+    // Logic for More Controller; user is shown the native list of tabs.
+    // This we want to keep as-is, we're not styling the ScrollView here,
+    // so let's pretend there isn't one
+    return NULL;
```

Comment:
Do not use `NULL` with Objective-C. Always prefer `nil`, unless you're interacting directly with C-style APIs, which is not the case here.